extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/runtime/OidcClientImpl.java

```diff
@@ -287,6 +287,9 @@ private Long getAccessTokenExpiresAtValue(String token, Object expiresInValue) {
             final long now = System.currentTimeMillis() / 1000;
             expiresAt = now + oidcConfig.accessTokenExpiresIn().get().toSeconds();
         }
+        if (oidcConfig.accessTokenExpiresInSkew().isPresent()) {
```

Comment:
but technically it is possible taht `expiresAt` is `null` here. why else would `if` condition handled such a scenario. I don't think you can do `+=`.

Reply:
@michalvavrik Good catch (that would be that Salesforce case where the binary access token without any expiry hints was sent), let me just add a null check there, thanks